src/cloudant/database.py

```diff
@@ -586,11 +590,8 @@ def missing_revisions(self, doc_id, *revisions):
         url = posixpath.join(self.database_url, '_missing_revs')
         data = {doc_id: list(revisions)}
 
-        resp = self.r_session.post(
-            url,
-            headers={'Content-Type': 'application/json'},
-            data=json.dumps(data)
-        )
+        resp = self.r_session.post(url, headers={'Content-Type': 'application/json'},
+                                   data=json.dumps(data))
```

Comment:
The problem is we either have to relax your pylintrc constraint of 1000 lines per file or, as I've done here, remove newlines.  I did not want to arbitrarily relax your testing standards so opted to flatten a few method calls as noted in my original comments.  Please advise.